_pages/cfp.md

```diff
@@ -26,17 +26,17 @@ All accepted submissions will be accompanied by a poster presentation. A number
-- Leveraging pretrained models of other modalities for time series.
-- Multimodal time series models.
-- Large scale evaluation benchmarks, live (continuously updating) datasets, synthetic data generation and data augmentation for time series.
-- Time series evaluation (e.g. analysis and development of metrics for (probabilistic) forecasting, multivariate forecasting, use-case motivated metrics etc.)
-- Real-world applications of large time series models.
+We invite submissions related to time series foundation models (TSFMs). Key topics include, but are not limited to:
+
+- **Building Foundation Models for Time Series Data**: The heterogeneity of time series data and tasks presents unique challenges in developing TSFMs. We welcome contributions exploring various design choices and improving our understanding of how these models scale with the amount and diversity of data.
+- **Analyzing Existing Pretrained Time Series Models**: TSFMs are often criticized for their black-box nature, especially compared to interpretable statistical models. We encourage submissions that analyze pretrained time series models to enhance our understanding of their learning processes.
+- **Critiques of Foundation Models for Time Series**: Contributions highlighting the limitations of TSFMs through theoretical analysis or systematic empirical evaluations are welcome.
+- **Faster and Better Inference Schemes for Autoregressive TSFMs**: Single-step autoregressive TSFMs are generally slower than multi-step models, such as those based on patching. We invite submissions comparing these techniques and developing methods to improve the inference speed and quality of autoregressive models.
+- **Leveraging Pretrained Models of Other Modalities for Time Series**: Recent studies show promise in adapting pretrained LLMs to specialized time series tasks. We seek to understand how design choices in leveraging these models—such as prompting techniques, adaptation methods, and fine-tuning—impact performance. We also seek to identify scenarios where these methods excel compared to training TSFMs from scratch, in terms of model capabilities, accuracy, and training and inference times.
+- **Multimodal Time Series Models**: Most time series models handle only numerical data, often providing a partial picture of the system of interest. In real-world settings, multiple modalities are available, and incorporating exogenous information, such as text, can enhance performance. We invite submissions exploring time series models that integrate information from other modalities.
+- **Large-Scale Time Series Datasets and Benchmarks**: The quality and quantity of publicly available time series data lag behind other modalities, such as text and vision. We welcome contributions of large-scale time series data and benchmarks, both general and domain-specific. We also invite methods for better synthetic time series generation and augmentation to address data challenges.
```

Comment:
Here, it'll be nice to include:
* A stress on foundation model benchmarks comparing various models
* Live datasets that continuously update; which I think is a trivial thing with time series datasets but I haven't seen papers on. Should be very useful for TSFM evaluation; avoids the memorization issues in evaluating them.

Reply:
- Done.
- If you think this is critical, please feel free to make an edit suggestion and I will accept it.

Reply:
Thanks! I don't think (2) is critical; was an idea. Maybe it's best to leave it general.